javascript/javascript1/week2/eventApplication.js

```diff
@@ -0,0 +1,10 @@
+const weekDays = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+const todaysDay = new Date();
+
+function getEventWeekday(days) {
```

Comment:
Nice and simple! 🤩 What would happen if I opened your page just before midnight and started using it after midnight? 🤔 Can you see what unexpected behavior might occur? And if so, how would you mitigate it? 😄

Reply:
I think I can understand the situation, but sorry, I don't know the answer... Let me have a try.

Reply:
It is a bit convoluted to spot the issue here, but imagine you had this script loaded onto a website and a user pressing a button would run `getEventWeekday(...)` and the user did not close the window for multiple days. What happens with `todaysDay` then? 😄